modules/base/rendering/renderablenodeline.cpp

```diff
@@ -125,7 +150,52 @@ RenderableNodeLine::RenderableNodeLine(const ghoul::Dictionary& dictionary)
     _lineWidth = p.lineWidth.value_or(_lineWidth);
     addProperty(_lineWidth);
 
-    addProperty(_opacity);
+    _start = p.startNode.value_or(_start);
+    addProperty(_start);
+
+    _end = p.endNode.value_or(_end);
+    addProperty(_end);
+
+    addProperty(_startOffset);
+    _startOffset.setExponent(12.f);
+
+    addProperty(_endOffset);
+    _endOffset.setExponent(12.f);
+
+    addProperty(_useRelativeOffsets);
+    _useRelativeOffsets.onChange([&]() {
+        SceneGraphNode* startNode = global::renderEngine->scene()->sceneGraphNode(_start);
+        SceneGraphNode* endNode = global::renderEngine->scene()->sceneGraphNode(_end);
+
+        if (!startNode) {
+            LERROR(fmt::format(
+                "Error when recomputing node line offsets for scene graph node '{}'. "
+                "Could not find start node '{}'", _identifier, _start.value()
+            ));
+            return;
+        }
+
+        if (!endNode) {
+            LERROR(fmt::format(
+                "Error when recomputing node line offsets for scene graph node '{}'. "
+                "Could not find end node '{}'", _identifier, _end.value()
+            ));
+            return;
+        }
+
+        if (_useRelativeOffsets) {
+            // Recompute previous offsets to relative values
+            double startBs = startNode->boundingSphere();
+            double endBs = endNode->boundingSphere();
+            _startOffset = startBs > 0.0 ? _startOffset / startBs : 0.0;
```

Comment:
Maybe provide some warning when it is set to zero because of no bounding sphere? I got confused the first time I tried it out with the transform nodes and not the planet nodes.